rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/admin/OpenehrAdminTemplateController.java

```diff
@@ -36,7 +36,7 @@
 @Api(tags = {"Admin", "Template"})
 @ConditionalOnProperty(prefix = "admin-api", name = "active")
 @RestController
-@RequestMapping(path = "/rest/openehr/v1/admin/template", produces = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_XML_VALUE})
+@RequestMapping(path = "/rest/admin/template", produces = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_XML_VALUE})
```

Comment:
Historically, the file names with `Openehr[...]` were used to distinguish between the EhrScape and openEHR API. Since you removed it from the openEHR API, it might make sense to rename and/or to move the file out of the `.../rest/openehr/...` scope now.

Reply:
Fixed